### PR TITLE
feat: move Loading element

### PR DESCRIPTION
**Pull request by lucianaDilizio**
Move Loading element to allow react render better while page is loading. 

To do: 
- [x] Delete general loading in subscriber-history screen
- [x] Move loading element into parts of the code that needs get info from the BE.
- [x] Do the same with subsciber-gdpr

https://cdn.fromdoppler.com/doppler-webapp/demo-build2452/#/reports/subscriber-history
https://cdn.fromdoppler.com/doppler-webapp/demo-build2452/#/reports/subscriber-gdpr

### CODE REVIEW

File: src/components/Reports/SubscriberHistory/SubscriberHistory.js [hunk 1]
```diff
@@ -209,109 +210,118 @@ const SubscriberHistory = ({
...
+                                            src={campaign.urlImgPreview}
+                                            alt={_('subscriber_history.alt_image')}
+                                          />
+                                        </div>
+                                      </a>
+                                    </div>
+                                  ) : (
                                     <a
-                                      href={`${reportsUrl}/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
+                                      href={`https://reports2.fromdoppler.com/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
```
Comment (cbernat) [inline src/components/Reports/SubscriberHistory/SubscriberHistory.js:276]: 🤔

**Comment by cbernat** [comment]
Could you fix the build to see how it works in cdn please?

File: src/components/Reports/SubscriberHistory/SubscriberHistory.js (hunk 1 above)
Comment (lucianaDilizio) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.js:276]: I have to make a rebase 😅

Comment (lucianaDilizio) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.js:276]: Or not... I don't know what happens here 😕

Comment (lucianaDilizio) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.js:276]: I didn't change anything there. Github mark this as a change but it's not.

**Comment by lucianaDilizio** [comment]
test this please

**Comment by lucianaDilizio** [comment]
@cbernat now you can see it https://cdn.fromdoppler.com/doppler-webapp/qa-build2451/#/reports/

File: src/components/Reports/SubscriberHistory/SubscriberHistory.js [hunk 2]
```diff
@@ -209,109 +210,118 @@ const SubscriberHistory = ({
...
+                  )}
                 </div>
               </div>
             </div>
           </div>
         </div>
       </section>
     </>
-  ) : (
-    <SafeRedirect to="/Lists/MasterSubscriber/" />
```
Comment (andresmoschini) [inline src/components/Reports/SubscriberHistory/SubscriberHistory.js:314]: It is fine moving the redirect inside so many nested divs?

Maybe you can move the redirect to the first lines and reduce the nesting. Please call me if I cannot express my idea.

**Comment by cbernat** [comment]
> @cbernat now you can see it https://cdn.fromdoppler.com/doppler-webapp/qa-build2451/#/reports/

Demo link here:
https://cdn.fromdoppler.com/doppler-webapp/demo-build2451/#/reports/subscriber-history

This is comming along nicely 
image

**Comment by cbernat** [comment]
🚀

File: src/components/Reports/SubscriberHistory/SubscriberHistory.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.js:278]: Mmmm, something happens here

Comment (lucianaDilizio) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.js:278]: I've just merged it 😢
This is something we have to change surely but I can do it in other PR with only refactors (without any visual changes0

**Pull request merged**